scripts/MRK17_Exp1_rk01.jmd

```diff
@@ -49,25 +52,28 @@ terms and associated correlation parameters in the RE structure.
 ## Model fit
 
 ```julia
-const HC = HelmertCoding();
-const contrasts = Dict(:F => HC, :P => HC, :Q => HC, :lQ => HC, :lT => HC);
+contrasts = merge(
+    Dict(nm => HelmertCoding() for nm in (:F, :P, :Q, :lQ, :lT)),
+    Dict(nm => Grouping() for nm in (:Subj, :Item)),
+);
 
 m1form = @formula (-1000/rt) ~ 1+F*P*Q*lQ*lT +
                               (1+F+P+Q+lQ+lT | Subj) +
                               (1+P+Q+lQ+lT | Item);
-cmplxLMM = @time fit(MixedModel, m1form, dat, contrasts=contrasts);
+cmplxLMM = fit(MixedModel, m1form, dat, contrasts=contrasts);
+cmplxLMM.PCA
 ```
 
 ## VCs and CPs
 
 We don't look at fixed effects before model selection.
 
 ```julia
-cmplxLMM.λ[1]
-cmplxLMM.λ[2]
-cmplxLMM.rePCA
+first(cmplxLMM.λ)
+```
+```julia
```

Comment:
These need to be two separate code blocks for the sake of the Jupyter notebooks.